uktrade/data-hub-frontend
# Remove the old company business details page

## Description of change

This PR is removing the company business details based on nunjucks with a React-based version introduced in https://github.com/uktrade/data-hub-frontend/pull/2267

## Test instructions

Go to `companies/COMPANY_ID/business-details` and check if everything works as before.
